storage/innobase/log/log0chkp.cc

```diff
@@ -1232,6 +1237,8 @@ static void log_update_limits_low(log_t &log) {
   ut_ad(log_limits_mutex_own(log));
 
   const lsn_t log_capacity = log_get_free_check_capacity(log);
+  if (log_capacity != log.max_checkpoint_age.load())
+    log.max_checkpoint_age.store(log_capacity);
```

Comment:
Shouldn't it be just
```suggestion
   log.max_checkpoint_age. = log_capacity;
```